rfd/0093-offline-access.md

```diff
@@ -0,0 +1,216 @@
+---
+authors: Tim Ross (tim.ross@goteleport.com)
+state: draft
+---
+
+# RFD 0093 - Robust Access to Nodes
+
+
+# Required Approvers
+
+* Engineering @zmb3 && (fspmarshall || codingllama)
+
+## What
+
+Ensure that users can access nodes via `tsh ssh` when there is intermittent or no connectivity to Auth.
+
+## Why
+
+Connectivity to the Auth server is a single point of failure which can result in not being able to access
+a resource. If a user already has already obtained a valid certificate (via `tsh login`) there are certain
+scenarios when a connection to Auth should not required to gain access to a node. This would reduce the burden
+on cluster admins when performing an Auth upgrade. On top of the potential downtime during the upgrade, any issues 
+found as a result of the upgrade require a rollback, which can further extend downtime and lack of connectivity.
+
+## Details
+
+### Auth Connection Required 
```

Comment:
I suppose Connect has the same characteristics as `tsh`, but should be mention it in a section of its own?